src/hotspot/share/utilities/concurrentHashTable.inline.hpp

```diff
@@ -1298,4 +1339,75 @@ inline bool ConcurrentHashTable<CONFIG, F>::
   return true;
 }
 
+template <typename CONFIG, MEMFLAGS F>
+class ConcurrentHashTable<CONFIG, F>::BucketsClaimer {
+  // Default size of _claim_size_log2
+  static const size_t DEFAULT_CLAIM_SIZE_LOG2 = 7;
+
+  ConcurrentHashTable<CONFIG, F>* _cht;
+  // The table is split into ranges, every increment is one range.
+  volatile size_t _next_to_claim;
+  size_t _claim_size_log2; // Log number of buckets in claimed range.
```

Comment:
For naming the constants containing log values (if kept), I would prefer if the existing style to put the `log2` in the front were kept.